cmd/genji/dbutil/dump_test.go

```diff
@@ -88,3 +88,65 @@ func TestDump(t *testing.T) {
 		})
 	}
 }
+
+func TestDumpSchema(t *testing.T) {
+	tests := []struct {
+		name   string
+		tables []string
+	}{
+		{"All tables", nil},
+		{"Selection of tables", []string{"test", "foo"}},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			db, err := genji.Open(":memory:")
+			require.NoError(t, err)
+			defer db.Close()
+
+			var want bytes.Buffer
+
+			getBuffer := func(table string) func(s string) {
```

Comment:
This was only necessary in the scope of the original test, it is not needed here

Reply:
It is necessary to keep the alphabetic order of the `tt.tables` else we should sort the slice of `tt.tables` manually.

Reply:
If we keep it, it would be useful to change l98 to `[]string{"tblA", "tblB", "tblC"}`, otherwise we won't have any output